.evergreen/config.yml

```diff
@@ -1120,6 +1120,39 @@ buildvariants:
   - name: publish-packages
     distros:
     - rhel70-small
+- name: amazon2023
+  display_name: "Amazon Linux 2023"
+  run_on: amazon2023.0-large
+  expansions:
+    has_packages: true
+    packager_distro: amazon2023
+    packager_arch: x86_64
+    node_lts_version: 16
+  tasks:
+  - build-and-test-and-upload
+  - build-and-test-shared-bson
+  - build-and-test-asan
+  - build-and-test-node
+  - test-java
```

Comment:
```suggestion
```

Suggest only running the `build-and-test-and-upload` and `publish-packages` tasks for the new variants. Those appear to be the only necessary tasks to publish packages.

It may be added maintenance burden for the Java and Node tasks to pass on all variants. Example: each distro may need to ensure all necessary Java and Node dependencies are available.

Comment applies to all new variants.